sample-programs/erc20/test-erc20.sh

```diff
@@ -0,0 +1,49 @@
+ALICE=SP2J6ZY48GV1EZ5V2V5RB9MP66SW86PYKKNRV9EJ7
```

Comment:
You should add `set -e` at the beginning of this script to ensure that any instance of `blockstack-core` exiting with non-zero exit status causes the test suite to abort.  Otherwise it's really hard to tell if all these tests passed.

Reply:
Also, it doesn't appear that there are any checks done in this test.  The test just writes data to stdout, and otherwise gives no indication of failure.  This must be fixed.

Reply:
> You should add set -e at the beginning of this script to ensure that any instance of blockstack-core exiting with non-zero exit status

If you look at the tests, I actually have some tests where abortions are expected (over-spending etc)

```
echo "-> SPENDER will send '10002' from BOB to ALICE - should fail"
blockstack-core local execute test-erc721.sqlite3 stx-erc721 transfer-from $SPENDER \'$BOB \'$ALICE 10002
echo "-> OPERATOR will send '10003' from CHARLIE to ALICE - should fail"
blockstack-core local execute test-erc721.sqlite3 stx-erc721 transfer-from $OPERATOR \'$CHARLIE \'$ALICE 10003
```

> The test just writes data to stdout, and otherwise gives no indication of failure. This must be fixed.

Agreed.
Writing tests was just painful with this `sh` script. I'll start leveraging @zone117x 's work this week for creating a complete suite of tests!!